### ISSUE TITLE
Description for Icons

### ISSUE THREAD

**Issue by MarcHaunschild** [w3c/wcag3#191]
In Germany we earlier had a test, that demanded textual description for icons, but there were a lot of people (a11y specialists) who criticized this test, because it differed from international rules (wcag, of course).
So the test (BITV-Test for those who know it) changed over the years to get closer to what is actually written in wcag. Although this makes sense, I think we lost a few things that were important for people with certain disabilities.
The reason for a textual description for icons is quite easy: I can't think of a single icon, that everybody would understand. Just take the icons here in this editor: a capital "A" and a bigger capital "A" together on one icon can stand for making texts bigger (or smaller), they mean nothing to people, who can't read, who don't know the latin alphabet or don't see at all. For those, that do not see, there is an aria-label - but what about the broad majority that is able to see? I would never have guessed, that it stands for "Add header text".
To make this short: I think there must be a textual description to make icons like these accessible to everybody.
These buttons here on the top of the editor are not accessible for people using a touch screen or a keyboard and there are not accessible for speech input (and don't have to, because SC 2.5.3 "Label in name" don't addresses graphical buttons, although they of course are user interface components. I think this should be changed also).
I think this is a problem for many people, not addressed by the WCAG yet and I suggest adding it in a future version.
